Sony9PinRemote/Types.h

```diff
@@ -349,6 +349,10 @@ struct Errors {
     bool b_buffer_overrun;
     bool b_framing_error;
     bool b_timeout;
+
+    Errors() {
```

Comment:
`memset` can be acceptable but brace initialization may be better for this like:

```C++
    bool b_unknown_cmd {false};
```

I will try it later. Please confirm if it works!

Reply:
Yes, too many `{false}`... lol But I prefer brace initializer so I will. Could you try it after that?